microservice/challenge-1/app42/frontend/src/services/UserService.js

```diff
@@ -0,0 +1,22 @@
+import axios from './Api';
+
+const get = () => {
+  return new Promise(resolve => {
```

Comment:
Is there any difference on writing 
```
return new Promise(resolve => {
    axios.get(`users`).then(getResponse => {
      resolve(getResponse.data);
    });
});
```
and this:
```
return axios.get(`users`).then(getResponse => getResponse.data);
```?

Reply:
Of course the second one is better.